src/pages/userPage/ActiveMembership/PlastDegree/FormAddPlastDegree.tsx

```diff
@@ -44,13 +54,22 @@ const FormAddPlastDegree = ({
       plastDegreeId: info.plastDegree,
       dateStart: info.datepickerStart._d,
       userId: userId,
+      
     };
     setVisibleModal(false);
     visiableDegree.current = false;
+    visiableCities.current = false;
+  
+    const cityMember1 : CityMember =  (await addFollowerWithId(info.city,userId)).data;
+    await toggleMemberStatus(cityMember1.id);
+
     await activeMembershipApi.postUserPlastDegree(userPlastDegreePost);
     handleAddDegree();
     form.resetFields();
     resetAvailablePlastDegree();
+   
+    if( UpdateData){  await UpdateData()};
```

Comment:
format code
if you want use condition statements in one line, consider using `ternary` operator